ksql-engine/src/main/java/io/confluent/ksql/engine/EngineContext.java

```diff
@@ -144,7 +144,8 @@ PreparedStatement<?> prepare(final ParsedStatement stmt) {
       throw e;
     } catch (final Exception e) {
       throw new KsqlStatementException(
```

Comment:
Maybe not in the PR, is it easier and safer to pass `stmt` to `KsqlStatementException` and use `getStatementText` inside it? In that way, there's no risk of passing wrong strings I guess.